opencti-platform/opencti-graphql/src/back.js

```diff
@@ -59,3 +60,22 @@ if (ENABLED_EVENT_LOOP_MONITORING) {
 // -- Start the platform
 // noinspection JSIgnoredPromiseFromCall
 platformStart().catch((reason) => logApp.error('Error occurs on platformStart', { reason }));
+// testJsonMapper({ externalUri: 'https://4.233.151.63:444' }, json, mispJsonMapper);
+
+// const trino = Trino.create({
+//   server: 'http://localhost:8080',
+//   catalog: 'tpcds',
+//   schema: 'sf1',
+//   auth: new BasicAuth('admin'),
+// });
+//
+// trino.query(
+//   'select * from customer'
+// ).then(async (iter) => {
+//   for await (const queryResult of iter) {
+//     console.log(`iter result ${(queryResult.data ?? []).length}`);
+//   }
+// });
+
+// testJsonMapper({ externalUri: 'https://4.233.151.63:444' }, json4, mispJsonMapper4);
+// jsonExecutor().then(() => jsonExecutor().then(() => jsonExecutor()));
```

Comment:
Do you want to keep this comment ? Is it interesting to keep that as test maybe ?